### DISCUSSION TITLE
Can I change the settings of Snapper to suit my need?

### DISCUSSION THREAD

**Discussion by Disil** [SpiralLinux/SpiralLinux-project#335] [Q&A] [answered]
After few months of using SpiralLinux and constantly installing/uninstalling software, I found that Snapper is too intrusive by having them creating snapshot everytime I do apt operations. Can I change it to make it not automatic like that, and instead take snapshot every day/week/month instead?

Also OOT but whenever I install/uninstall something, at the end of the apt terminal output there is always a text that says "Illegal snapshot." I wonder what that means.

**Comment by geckolinux** [comment, marked as answer]
Hi there,

> I found that Snapper is too intrusive by having them creating snapshot everytime I do apt operations. Can I change it to make it not automatic like that

Yes, in `/etc/default/snapper` you can set `DISABLE_APT_SNAPSHOT="yes"` .

> and instead take snapshot every day/week/month instead?

You could setup a manual cron job to run `snapper -c root create -c number` or  `snapper -c root create -c timeline`. If you want the automatic cleanup algorithm that is shown in **Snapper GUI**  to maintain a fixed number of snapshots then use `-c number` , or if you want it to retain X number of daily/monthly/weekly snapshots you could use `-c timeline` .

> Also OOT but whenever I install/uninstall something, at the end of the apt terminal output there is always a text that says "Illegal snapshot." 

Not sure about this one, you could ask on a more technical forum.

**Reply by Disil** [reply]
Could I disable this configuration from Snapper and use the one provided by Timeshift instead??

**Reply by geckolinux** [reply]
Hmm, no, Timeshift won't work with that subvolume layout.
